plotters-svg/src/svg.rs

```diff
@@ -333,21 +456,18 @@ impl<'a> DrawingBackend for SVGBackend<'a> {
         if style.color().alpha == 0.0 {
             return Ok(());
         }
-        self.open_tag(
-            SVGTag::Polygon,
-            &[
-                ("opacity", &make_svg_opacity(style.color())),
-                ("fill", &make_svg_color(style.color())),
-                (
-                    "points",
-                    &path.into_iter().fold(String::new(), |mut s, (x, y)| {
-                        write!(s, "{},{} ", x, y).ok();
-                        s
-                    }),
-                ),
-            ],
-            true,
-        );
+        let mut attrwriter = self.open_tag(SVGTag::Polygon);
+        attrwriter
+            .write_key("opacity")
+            .write_value(style.color().alpha);
+        attrwriter
+            .write_key("fill")
+            .write_value(make_svg_color(style.color()));
+        attrwriter
+            .write_key("points")
+            .write_value(FormatEscapedIter(path.into_iter()));
```

Comment:
This can't be correct given that 446 does it differently. Plus in line 446 there might be a whitespace missing at the end of each point.